tools/harness-automation/autothreadharness/runner.py

```diff
@@ -106,10 +106,17 @@ def add_result(self, test, passed, error=None):
             test (TestCase): The test just run
             passed (bool): Whether the case is passed
         """
+        if str(test.__class__.__name__) in self.result and 'fails' in self.result[str(test.__class__.__name__)]:
+            fails = int(self.result[str(test.__class__.__name__)]['fails'])
+        else:
+            fails = 0
+        if passed is False:
+            fails += 1
         self.result[str(test.__class__.__name__)] = {
             'started': self.started,
             'stopped': time.strftime('%Y-%m-%dT%H:%M:%S'),
             'passed': passed,
+            'fails': fails,
```

Comment:
Are the tests supposed to stop once we get an `Pass` ?
If we run 4 tests: 2 fail then 2 pass, we are going to get the result:
```
{
    ...
    'fails': 2, 
    'passed': True,
    ...
}
```

So we don't know how many pass we got. 
Why not also save the number of `Pass` we got?

Reply:
For current test strategy, once passed, automation tools will move to test next case.